Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
What does it take to add Android as a platform for the compiler to understand here? That may be a prereq to anyone doing anything meaningful on Android in the first place.